.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{yml,yaml,json}]
+indent_style = space
```

Comment:
This is a well-recognised standard for configuring linefeed and tab style, we use it in a lot of places and its very useful for avoiding mixed whitespace errors among other common pain points.

Reply:
I meant the whole file, not these two lines specifically. But regardless, what pain points does this specifically solve that aren't taken care by balena-lint. Why should the project force its opinion on a contributor on how should they should *go about* fulfilling balena-lint's rules?

Reply:
Just trying to understand as it took me by surprise that we're adding this file and couldn't (still can't, honestly) see the point given we have balena-lint configured.

I had a quick look at a few repositories and indeed a small percentage includes .editorconfig, and of these most were different, primarily agreeing on the use of tabs instead of spaces. Given that I configure my editors to infer settings from the file, I never noticed the effect of these config files so we may as well leave it in and it's not a problem at all to me. Just curious.